website/src/app/layout.tsx

```diff
@@ -41,6 +46,6 @@ export default function RootLayout({
         />
       </head>
       <body className="flex min-h-full flex-col">{children}</body>
-    </html>
+    </html >
```

Comment:
You doesn't seems to not a formater. You can use a formatter to make the code pretty. You can use [prettier](https://prettier.io/) for that.

I will make the style of the code consistant between developer. It helps to reduce the cognitive complexity by using the same formatting between developer, no more needs to understand the formatting from other developers.

It will also help to reduce git conflicts.